contracts/modules/TokenExchanger.sol

```diff
@@ -33,139 +35,238 @@ contract TokenExchanger is OnlyOwnerModule {
 
     // Mock token address for ETH
     address constant internal ETH_TOKEN_ADDRESS = 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
+    // Signatures of Paraswap's trade methods
+    bytes4 constant internal MULTISWAP = 0xcbd1603e;
```

Comment:
Maybe useful to have the computation of the signatures in the comment (`bytes4(keccak256(.....)`) so we know exactly what is called.

Reply:
Done